src/native/external/LinuxTracepoints/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
This should be included in https://github.com/dotnet/runtime/blob/main/THIRD-PARTY-NOTICES.TXT

Reply:
Added